src/index.js

```diff
@@ -115,6 +116,8 @@ async function run(params) {
     params.__ow_query = query;
   }
   const url = `${params.__ow_path.substring(1)}?${params.__ow_query || ''}`;
+  const lvls = url.split('.');
+  params.kind = lvls[lvls.length - 2];
```

Comment:
Looking at the examples in the test, I'm thinking we should append potential 3rd level domains, unless it's `www`: e.g. a hostname `spark.adobe.com` would result in `embed-adobe-spark`, but `www.slideshare.com` stays `embed-slideshare`. This would allow us to differentiate between multiple services from the same 2nd level domain. Wdyt?

Reply:
I like this idea! IT's included in the next commit :D